components/esp-tls/esp_tls.c

```diff
@@ -594,6 +594,24 @@ esp_err_t esp_tls_cfg_server_session_tickets_init(esp_tls_cfg_server_t *cfg)
 #endif
 }
 
+#ifdef CONFIG_ESP_TLS_SERVER
```

Comment:
This `#if` is not necessary. I think the code is already under the macro of the same name.